ui/app/AppLayouts/Communities/panels/SharedAddressesPanel.qml

```diff
@@ -0,0 +1,679 @@
+
+    component PanelIcon: StatusRoundIcon {
+        Layout.preferredWidth: 40
+        Layout.preferredHeight: 40
+        Layout.alignment: Qt.AlignTop
+        asset.name: {
+            switch (permissionType) {
+            case PermissionTypes.Type.Admin:
+                return "admin"
+            case PermissionTypes.Type.Member:
+                return "communities"
+            default:
+                return "channel"
+            }
+        }
+        radius: height/2
+    }
+
+    component PanelHeading: StatusBaseText {
+        Layout.fillWidth: true
+        elide: Text.ElideRight
+        font.weight: Font.Medium
+        text: {
+            switch (permissionType) {
+            case PermissionTypes.Type.Admin:
+                return qsTr("Become an admin")
+            case PermissionTypes.Type.Member:
+                return qsTr("Join %1").arg(root.communityName)
+            default:
+                return modelData // TODO display channel name https://github.com/status-im/status-desktop/issues/11481
+            }
+        }
+    }
+
+    component SinglePermissionFlow: Flow {
+        width: parent.width
+        spacing: Style.current.halfPadding
+        Repeater {
+            model: HoldingsSelectionModel {
+                sourceModel: holdingsListModel
+                assetsModel: root.assetsModel
+                collectiblesModel: root.collectiblesModel
+            }
+            delegate: Row {
+                spacing: 4
+                StatusRoundedImage {
+                    anchors.verticalCenter: parent.verticalCenter
+                    width: 16
+                    height: 16
+                    image.source: model.imageSource
+                }
+                StatusBaseText {
+                    anchors.verticalCenter: parent.verticalCenter
+                    font.pixelSize: Theme.tertiaryTextFontSize
+                    text: model.text
+                    color: model.available ? Theme.palette.successColor1 : Theme.palette.directColor1
+                }
+            }
+        }
+    }
+
+    component PermissionPanel: Control {
+        id: permissionPanel
+        property int permissionType: PermissionTypes.Type.None
+        property var permissionsModel
+
+        readonly property bool tokenCriteriaMet: overallPermissionRow.tokenCriteriaMet
+
+        visible: permissionsModel.count
+        Layout.fillWidth: true
+        padding: d.absLeftMargin
+        background: PanelBg {}
+        contentItem: RowLayout {
+            spacing: Style.current.padding
+            PanelIcon {}
+            ColumnLayout {
+                Layout.fillWidth: true
+                Layout.fillHeight: true
+                PanelHeading {}
+                Rectangle {
+                    Layout.fillWidth: true
+                    Layout.preferredHeight: grid.implicitHeight + grid.anchors.margins*2
+                    border.width: 1
+                    border.color: Theme.palette.directColor7
```

Comment:
Looks like the border color should be `baseColor2`. Same for all permission cards and the Permissions scrollview.
https://www.figma.com/file/17fc13UBFvInrLgNUKJJg5/Kuba⎜Desktop?node-id=31461%3A564844&mode=dev